### DISCUSSION TITLE
Garmin RVR315 not able to detect objects coming from left and right

### DISCUSSION THREAD

**Discussion by mamtasiyak** [maca88/SmartBikeLights#74] [General]
Folks I am getting data using ANT+ protocol to detect objetcs on road using Garmin RVR315 and it is not able to identify objects coming from left and right, it always says for all the objects approaching from behind even if it is coming from left or right. Is it working fine for everyone else? Is it only me who is facing this issue? What do I do?

**Comment by maca88** [comment]
Hi, I did some quick tests my on Varia RTL515 and I am also getting "THREAT_SIDE_NO_SIDE" every time. It seems like that Varia does not support [threatSide](https://developer.garmin.com/connect-iq/api-docs/Toybox/AntPlus/RadarTarget.html#threatSide-var) attribute.

**Comment by mamtasiyak** [comment]
Thanks :)
